official/resnet/resnet_run_loop.py

```diff
@@ -244,7 +255,7 @@ def exclude_batch_norm(name):
 
   # Add weight decay to the loss.
   l2_loss = weight_decay * tf.add_n(
-      [tf.nn.l2_loss(v) for v in tf.trainable_variables()
+      [tf.nn.l2_loss(tf.cast(v, tf.float32)) for v in tf.trainable_variables()
```

Comment:
Comment as to why we must explicitly cast here and not elsewhere?

Reply:
Done.